_posts/2017-04-09-maryam-critique.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title:  "Critique of New York Times 'Death in Syria' Graphic"
+date:   2017-04-09
+author: Maryam Saleh
+---
+
+[This](https://www.nytimes.com/interactive/2015/09/14/world/middleeast/syria-war-deaths.html?_r=0) New York Times data visualization on the different ways people in Syria are dying is great in that it helps make sense of a conflict that is obscure in the minds of many Americans. While I liked the use of dots to represent the number of Syria’s dead, I can’t help but wonder if there was a way to make the data even more relatable. For example, it would have been helpful to include a side-by-side comparison to deaths from another conflict or major incident.
```

Comment:
You should link a few more words to make it easier for a reader to identify what a link is about and encourage them to click it.